app/views/buses/arrivals.html.erb

```diff
@@ -0,0 +1,13 @@
+<h1>Buses</h1>
+
+<% @arrivals.each do |stop| %>
```

Comment:
If the returned list is empty we might want to display a message, e.g. buses not found or error message from the backend